backend/src/models/event.test.ts

```diff
@@ -101,38 +116,39 @@ describe("get", function () {
 
 describe("update", function () {
   const updateData = {
-    buskerId: "b1",
+    buskerId: "",
     title: "newTitle",
     type: "NewType",
     coordinates: { lat: 45, lng: 45 },
   };
 
   test("works", async function () {
```

Comment:
Need a test for updating with partial data.

Reply:
Added a test for partial data update, which required changes to the `EventData` interface in the `event` model. See https://github.com/diogo-botelho/busk-app/pull/26/commits/f0596a7c9dcd8f6e319f7a8b8b2fbb483ce3c281.